include/mbedtls/config.h

```diff
@@ -2545,6 +2551,15 @@
 //#define MBEDTLS_PLATFORM_NV_SEED_READ_MACRO   mbedtls_platform_std_nv_seed_read /**< Default nv_seed_read function to use, can be undefined */
 //#define MBEDTLS_PLATFORM_NV_SEED_WRITE_MACRO  mbedtls_platform_std_nv_seed_write /**< Default nv_seed_write function to use, can be undefined */
 
+/* Threading/mutex operations */
+/* To use threading operations MBEDTLS_THREADING_C must be defined */
+//#define MBEDTLS_MUTEX_INIT_MACRO        mbedtls_mutex_init
+//#define MBEDTLS_MUTEX_FREE_MACRO        mbedtls_mutex_free
+//#define MBEDTLS_MUTEX_LOCK_MACRO        mbedtls_mutex_lock
+//#define MBEDTLS_MUTEX_UNLOCK_MACRO      mbedtls_mutex_unlock
+
+//#define MBEDTLS_MUTEX_INITIALIZER       { NULL }
```

Comment:
I think this macro allows the static initialization of mutexes. This is a pthread feature and this is probably there was a MUTEX_INIT macro in that implementation of the threading API. However, I am not sure whether this static initialisation feature is common in other threading libraries. I would suggest removing this macro, or at least explaining what it is meant to do. At the moment it is not clear what its the function without looking into the source code.